src/Algebra/Construct/Flip/Op.agda

```diff
@@ -0,0 +1,241 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Flipping the arguments of a binary operation preserves many of its
+-- algebraic properties.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Algebra.Construct.Flip.Op where
+
+open import Algebra
+open import Data.Product using (swap)
```

Comment:
Can we be consistent between `Product` and `Sum` and import them both qualified? i.e. `import Data.Product as Prod`

Reply:
Fixed.